client/src/components/cart/Cart.jsx

```diff
@@ -97,38 +101,6 @@ function Cart() {
   };
 
 
-  async function createStripeCheckoutSession() {
```

Comment:
@ilkckn can you please explain why you're deleting the checkoutStripeSession function?